hugo/content/docs/getting_started.md

```diff
@@ -0,0 +1,263 @@
+---
+title: 'Getting Started'
+weight: 200
+---
+Our 'Getting Started' example consists of a simple application that displays a list of tasks, their status, and the relationship between them in a HTML page.
+
+The main steps to integrate Sprotty into our application are as follows:
+1. [Set-up](#setting-up-our-application) your application.
+2. [Define your model](#define-your-model) by creating subclasses of `SModelElement`.
+3. [Implement views](#implement-views) to generate SVGs for each type of model element.
+4. [Configure the diagram](#configure-the-diagram) through dependency injection.
+5. [Connect to a model source](#connect-to-a-model-source), either local or remote
+
+## Setting-up our application
```

Comment:
Unsure whether we should capitalize all proper nouns in the headers. I recommended this in Jonah's PRs, but now I'm not so sure. Either way, it would be good to make sure our headers are fairly uniform in style across doc pages.